src/docs/product/releases/index.mdx

```diff
@@ -42,7 +42,7 @@ Each release links to one project. If a release has multiple projects, Sentry wi
 
 _Release health_ provides insight into the impact of crashes and bugs as it relates to your user's experience and reveals trends with each new issue. Monitor [Release Health](/product/releases/health/) by observing user adoption, usage of the application, percentage of crashes, and session data.
 
-You can view Release Health data either from the Issue Details page by selecting the commit ID listed under **Last Seen** or from the Releases homepage.
+You can view Release Health data either from the <SandboxLink scenario="oneIssue" projectSlug="react">Issue Details</SandboxLink> page by selecting the commit ID listed under **Last Seen** or from the Releases homepage.
```

Comment:
Same comment here about linking to other topics. Strongly recommend the Sandbox link be removed here.